recipe-server/client/control_new/components/forms/TrimWhitespaceFormItem.js

```diff
@@ -0,0 +1,34 @@
+import autobind from 'autobind-decorator';
+import { message } from 'antd';
+import React from 'react';
+
+import FormItem from 'control_new/components/forms/FormItem';
+
+@autobind
+export default class TrimWhitespaceFormItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.shouldNotify = true;
```

Comment:
I think this should be more React-y and use `state` instead of an instance variable, then we would not need the constructor, only `state = { shouldNotify: true, };`

Reply:
I advised @marcrowo to use an instance variable since `shouldNotify` doesn't affect the rendering of the component, only event handling. Using state for this would produce spurious calls to `render`.

Though if we are removing the messages, it doesn't matter either way.